### PR TITLE
docs: add getting started and code push sections

### PR DESCRIPTION
**Pull request by felangel**
## Status

**READY**

## Description

- docs: add getting started and code push sections

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [X] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 ---
 sidebar_position: 1
+title: 📚 Overview
 ---
 
-# Overview 📚
+# Overview
 
-Welcome to the Shorebird docs.
+Welcome to Shorebird 👋
```
Comment (felangel) [inline docs/index.md:8]: Will come back to this in a subsequent PR.

File: docs/code_push/_category_.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "label": "☁️ Code Push",
+    "position": 3,
+    "link": {
+      "title": "Code Push",
+      "slug": "code-push",
+      "type": "generated-index",
+      "description": "Learn everything you need to add over the air updates (code push) to your Flutter application."
```
Comment (eseidel) [inline docs/code_push/_category_.json]: I'd probably reverse the parenthetical "to add code push (over the air updates)" 🤷   The point of using "code push" is mostly to piggy back on the naming used by RN.  e.g. https://microsoft.github.io/code-push/ which might be a term users have heard. 🤷

File: docs/code_push/build.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 5
+title: 🔨 Build
+description: Learn how to generate release builds of your Flutter app.
+---
+
+# Build your Shorebird App
+
+You can use `shorebird build` to build a release version of your app including code push:
```
Comment (eseidel) [inline docs/code_push/build.md:9]: `release` is generally the main flow, no?

**Review by eseidel — COMMENTED** [review]
I didn't read it all the way (yet).  Is it staged somewhere we could see it that way?

File: docs/code_push/build.md (hunk 3 above)
Comment (felangel) [reply docs/code_push/build.md:9]: Yeah build is last in this section.

**Comment by felangel** [comment]
> I didn't read it all the way (yet). Is it staged somewhere we could see it that way?

GitHub actions is down atm so the quickest way to see it would be to check out the branch and run it locally.

**Review by eseidel — APPROVED** [review]
I didn't review this in detail.  I think we should just land this and I will wordsmith and send you a PR. :)

**Pull request merged**